.github/workflows/ci.yml

```diff
@@ -32,17 +32,21 @@ jobs:
         run: yarn lint
       - name: Unit tests
         run: yarn test:ci
-      - name: Build frontend
-        run: yarn build
 
       - name: Compatibility check
         run: npx @grafana/levitate@latest is-compatible --path src/module.ts --target @grafana/data,@grafana/ui,@grafana/runtime
 
-      - name: Upload package for e2e tests
+      - name: Build frontend
+        id: build
+        uses: ./.github/build/
+        with:
+          build_identifier: dev.${{ github.run_id }}
```

Comment:
Have you thought about using a short commit sha here?

Reply:
we can if you want. but you can build twice for one commit, and the run id will increment (i hope)

Reply:
fine for me.

Reply:
would keept it as is, but we can adapt later on if you want.